src/components/KMultiselect/KMultiselect.vue

```diff
@@ -905,9 +905,13 @@ watch(() => props.items, (newValue, oldValue) => {
 }, { deep: true, immediate: true })
 
 const numericWidth = ref<number>(300)
-onMounted(() => {
-  // @ts-ignore
+const setNumericWidth = (): void => {
   numericWidth.value = multiselectRef.value?.clientWidth || 300
+}
+
+const resizeObserver = ref()
+onMounted(() => {
+  resizeObserver.value = new ResizeObserver(setNumericWidth).observe(multiselectRef.value as HTMLDivElement)
```

Comment:
suggestion: It's better to add `unobserve` on `onUnmounted` to prevent listeners after component is onUnmounted.